<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Co2Form</class>
 <widget class="QWidget" name="Co2Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <widget class="mold::InvisibleClickableButton" name="myLovelyButton">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>175</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="mold::RoomHeaderForm" name="header" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="mold::StatusBarForm" name="statusBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>215</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="co2_value">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="co2_unit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>104</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">ppm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="co2_rating">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>104</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>normal</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="co2_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>53</y>
     <width>320</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">CO₂</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="mold::Co2ProgressWidget" name="co2_render" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>41</y>
     <width>320</width>
     <height>10</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="warning_mold">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>151</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;WARNUNG!&lt;/span&gt; Nächste Lüftung&lt;br/&gt;in: 50 min&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="space">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>320</width>
     <height>1</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mold::InvisibleClickableButton</class>
   <extends>QPushButton</extends>
   <header>gui/InvisibleClickableButton.hpp</header>
  </customwidget>
  <customwidget>
   <class>mold::StatusBarForm</class>
   <extends>QWidget</extends>
   <header>gui/StatusBarForm.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mold::RoomHeaderForm</class>
   <extends>QWidget</extends>
   <header>gui/RoomHeaderForm.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mold::Co2ProgressWidget</class>
   <extends>QWidget</extends>
   <header>gui/Co2ProgressWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
